test/unittest/datasets/data_producer_common_tests.cpp

```diff
@@ -24,7 +24,6 @@ createSample(const std::vector<nntrainer::TensorDim> &input_dims,
   using namespace nntrainer;
   auto populate_tensor = [](const TensorDim &dim) {
     Tensor t(dim);
-    t.updateBatch(1);
```

Comment:
could you let it be disabled? I'll fix it.

Reply:
removed.